docs/source/augly.audio.rst

```diff
@@ -0,0 +1,53 @@
+augly.audio package
+===================
+
+Submodules
+----------
+
+augly.audio.composition module
```

Comment:
Just to make sure I understand: will we have to manually add any new modules we add later on to these files? Is there any way to make sphinx discover modules automatically? (It's ok if not, but I want to undertstand if these manual steps are 100% necessary). If there **are** manual steps to be done when we add to AugLy, please document those steps in your comment on this PR :)

Reply:
@zpapakipos Sphinx will do it automatically with the `sphinx-apidoc -o ./source ../augly` command.